tests/test_ge_utils.py

```diff
@@ -139,6 +139,35 @@ def test_validate_using_data_context(
     assert res.statistics["evaluated_expectations"] == 2
 
 
+def test_validate_using_newstyle_data_context(
+    dataset, data_context_v3_parameterized_expectation_suite
+):
+    # Before running, the data context should not have compiled parameters
+    assert (
+        data_context_v3_parameterized_expectation_suite._evaluation_parameter_dependencies_compiled
+        is False
+    )
+    with pytest.warns(
+        Warning, match=r"This configuration object was built using version"
+    ):
+        res = ge.validate(
```

Comment:
This looks odd to me.  Unless something changed, the `validate()` in `great_expectations/util.py` is the Pre-GE013 style.  So what is being tested here?  I would assume that we want to test `Validator.validate()` instead.  Clearly this also calls for a decision on what to do about the Legacy `validate()`, mentioned above.  My proposal would be to make it work seamlessly for both 013 and 012 versions (analogously to what `DataContext.load_batch()`) does.